extensions/pom.xml

```diff
@@ -805,7 +805,7 @@
         </exclusion>
       </exclusions>
     </dependency>
-    
+
```

Comment:
File has no (non-whitespace) changes and so should not be part of the PR.

Reply:
addressed